buildkit/pgrouting.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+name: pgrouting
+version: "3.5.1"
+homepage: https://github.com/pgRouting/pgrouting
+repository: https://github.com/pgRouting/pgrouting
+source: https://github.com/pgRouting/pgrouting/releases/download/v3.5.1/pgrouting-3.5.1.tar.gz
+description: pgRouting extends the PostGIS/PostgreSQL geospatial database to provide geospatial routing and other network analysis functionality.
+licence: GPL-2.0 AND BSL-1.0
```

Comment:
I think this may cause some issues for the front-end. I had a look at the license situation and `GPL-2.0-or-later` probably is the "best" license to summarize the situation here, BSL is a very permissive license. @owenthereal thoughts?

Reply:
The license is summarized [here](https://github.com/pgRouting/pgrouting/tree/main#license). I think GPL-2.0-or-later is okay because it claims most features are under this license.